gradle/libs.versions.toml

```diff
@@ -7,7 +7,7 @@ junit = "4.13.2"
 kotlin = "1.6.21"
 kotlinpoet = "1.12.0"
 ktlint = "0.46.0"
-mavenplugin = "0.22.0"
+mavenplugin = "0.24.0"
```

Comment:
Bump to a newer plugin. We can't bump to the latest without also bumping to a newer version of the Kotlin plugin (which isn't very straightforward). I'd like to start with this to get a release out.